### DISCUSSION TITLE
Map source not loading

### DISCUSSION THREAD

**Discussion by chrisnchips** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#755] [Q&A] [answered]
I have begun to set up the card. As Ecovacs does not work with map extractor I have used a screenshot but it does not show up on my dashboard -just the little icon. Where have I gone wrong?

![Suction_Map](https://github.com/user-attachments/assets/27f65a3c-8a04-4761-a4ff-c50888ee164c)
![image](https://github.com/user-attachments/assets/95df808f-2c55-4ad4-bc17-9eeb1b7fe39f)
![image](https://github.com/user-attachments/assets/b1e483f1-2a70-4bff-b419-f4985beb6864)

**Comment by sca075** [comment]
@chrisnchips it looks you setup the wrong path for your image. You need to point to the location of the screenshot.

**Reply by PiotrMachowski** [reply]
Exactly, path should be similar to `/local/community/...`

**Reply by chrisnchips** [reply]
is there a best path to place the file?

**Reply by PiotrMachowski** [reply, marked as answer]
You should put the image somewhere in `/config/www` or its subfolders. HA will make it available under `/local` path.

Examples (`<file location>` -> `<image url>`):
`/config/www/images/vacuum_map.png` -> `/local/images/vacuum_map.png`
`/config/www/vacuum_map.png` -> `/local/vacuum_map.png`

**Reply by chrisnchips** [reply]
Fantastic! it was exactly this and I can move onto cropping, thanks :)

**Comment by FoxXxHater** [comment]
How can i use the image entity from the ecovas intergration itself?

**Reply by PiotrMachowski** [reply]
You can just provide it instead of the camera entity (you have to do it in yaml editor)

**Reply by FoxXxHater** [reply]
And how could i do this?
Bcause if i write this, the map couldnt be loaded... 

![image](https://github.com/user-attachments/assets/4c78f46f-7edd-44e8-9032-5c7bb00fdf06)

**Reply by PiotrMachowski** [reply]
```yaml
map_source:
  camera: image.ecovacs_t30_pro_karte
```

**Reply by FoxXxHater** [reply]
Ahhhh, thanks... it works
